src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_grand_champions.cpp

```diff
@@ -16,577 +16,1263 @@
-    SPELL_SHIELD                    = 66482,
+    // Vehicle
+    SPELL_CHARGE_GRAND_CHAMP            = 68321,
+    SPELL_CHARGE_LESSER_CHAMP           = 63010,
+    SPELL_SHIELD_BREAKER                = 68504,
+    SPELL_SHIELD_GRAND_CHAMP            = 62719,
+    SPELL_SHIELD_LESSER_CHAMP           = 64100,
+    SPELL_TRAMPLE_DUMMY_AURA            = 67870, // triggers trample aura every 1 second, used by all vehicles
+    SPELL_TRAMPLE_AURA                  = 67868, // actual trample stun
+
+    // General
+    SPELL_ACHIEVEMENT_CREDIT            = 68572, // dummy spell for achievement credit (IDs 4297, 4298, 3778, 4296)
+    SPELL_KNEEL                         = 68442,
 
     // Marshal Jacob Alerius && Mokra the Skullcrusher || Warrior
-    SPELL_MORTAL_STRIKE             = 68783,
-    SPELL_MORTAL_STRIKE_H           = 68784,
-    SPELL_BLADESTORM                = 63784,
-    SPELL_INTERCEPT                 = 67540,
-    SPELL_ROLLING_THROW             = 47115, //not implemented in the AI yet...
+    SPELL_MORTAL_STRIKE                 = 68783,
+    SPELL_BLADESTORM                    = 67541,
+    SPELL_INTERCEPT                     = 67540,
+    SPELL_ROLLING_THROW                 = 67546,
 
     // Ambrose Boltspark && Eressea Dawnsinger || Mage
-    SPELL_FIREBALL                  = 66042,
-    SPELL_FIREBALL_H                = 68310,
-    SPELL_BLAST_WAVE                = 66044,
-    SPELL_BLAST_WAVE_H              = 68312,
-    SPELL_HASTE                     = 66045,
-    SPELL_POLYMORPH                 = 66043,
-    SPELL_POLYMORPH_H               = 68311,
+    SPELL_FIREBALL                      = 66042,
+    SPELL_BLAST_WAVE                    = 66044,
+    SPELL_HASTE                         = 66045,
+    SPELL_POLYMORPH                     = 66043,
 
     // Colosos && Runok Wildmane || Shaman
-    SPELL_CHAIN_LIGHTNING           = 67529,
-    SPELL_CHAIN_LIGHTNING_H         = 68319,
-    SPELL_EARTH_SHIELD              = 67530,
-    SPELL_HEALING_WAVE              = 67528,
-    SPELL_HEALING_WAVE_H            = 68318,
-    SPELL_HEX_OF_MENDING            = 67534,
+    SPELL_CHAIN_LIGHTNING               = 67529,
+    SPELL_EARTH_SHIELD                  = 67530,
+    SPELL_HEALING_WAVE                  = 67528,
+    SPELL_HEX_OF_MENDING                = 67534,
 
     // Jaelyne Evensong && Zul'tore || Hunter
-    SPELL_DISENGAGE                 = 68340, //not implemented in the AI yet...
-    SPELL_LIGHTNING_ARROWS          = 66083,
-    SPELL_MULTI_SHOT                = 66081,
-    SPELL_SHOOT                     = 65868,
-    SPELL_SHOOT_H                   = 67988,
-
-    // Lana Stouthammer Evensong && Deathstalker Visceri || Rouge
-    SPELL_EVISCERATE                = 67709,
-    SPELL_EVISCERATE_H              = 68317,
-    SPELL_FAN_OF_KNIVES             = 67706,
-    SPELL_POISON_BOTTLE             = 67701
+    SPELL_DISENGAGE                     = 68339, // 68340 is correct disengage but spell needs to be scripted
```

Comment:
i agree with the comment, `68340` needs to be scripted if it's the correct spell...